### DISCUSSION TITLE
Lync 2013 SDK for desktop apps .NET Core 3.1 compatibility

### DISCUSSION THREAD

**Discussion by Sybren-** [community/community#21504] [Other Feature Feedback, Questions, & Ideas]
<p>Is the Lync 2013 SDK for desktop apps .NET Core compatible? I couldn’t find any information about this in the docs.<br>
If it’s not compatible, is there a newer version of the SDK which integrates with Lync and Skype for Business and which supports .NET Core 3.1?</p>

**Comment by fire-eggs** [comment, marked as answer]
<p>Lync 2013 SDK pre-dates the entire .NET Core effort by years. You’d need a Lync SDK update which was post-2019 to be .NET Core 3.1 compatible.</p>
<p>In short: No, and No.</p>

**Comment by Sybren-** [comment]
<p>Do you have a link of that Lync SDK update?</p>
